<?xml version="1.0"?>

<launch>
    <param name="meshPath" value="$(find husky_cortex)/data/planehole.obj" />
    <param name="initLocation" value="-241. 5.0 -193." />
    <param name="velocity" value="100." />
    <param name="coverRange" value="30." />
    <param name="colorScheme" value="4"/>

    <param name="normDThreshold" value="50."/>
    <param name="bndDThreshold" value="20."/>

    <param name="baseTexture" value="$(find husky_cortex)/data/paper.png"/>

    <node name="cortexWorld" pkg="husky_cortex" type="CortexWorld" output="screen"/> 

</launch>
